spec/languages/lambda/convertAST-test.js

```diff
@@ -0,0 +1,61 @@
+import convertAST from 'codemirror-blocks/languages/lambda/convertAST';
+
+describe('the convertAST function for lambda to codemirror-blocks', function() {
+
+  it("should convert number to literal", function(){
+    this.AST = { type: "num", value: 123.5 };
```

Comment:
To simplify the code a bit in these tests, I would just not use `this.` but rather use `var`. i.e.

```javascript
var AST = {...};
var cAST = {...};
```